arch/Dockerfile

```diff
@@ -0,0 +1,66 @@
+## Emacs, make this -*- mode: sh; -*-
+
+FROM archlinux/base:latest
+
+## Set a default user. Available via runtime flag
+RUN useradd docker
+
+RUN pacman -Syu --noconfirm
+
+# installed in archlinux:base --
+# ca-certificates curl bzip2 findutils pcre readline xz zlib
+RUN pacman -S --noconfirm \
+	gcc-fortran \
+	less \
+	jdk8-openjdk  \
+	cairo \
+	ghostscript \
+    icu \
+	libjpeg-turbo \
+	pango \
+	libpng \
+	libtiff \
+	libx11 \
+    libxmu \
+	libxt \
+	make \
+	subversion \
+	tcl \
+	texinfo \
+	texlive-core \
+	texlive-fontsextra \
+	tk \
+	unzip \
+	xorgproto \
+	tar \
+	which \
+	xorg-server-xvfb
+
+
+RUN pacman -S --noconfirm valgrind
+
+RUN pacman -S --noconfirm qpdf
+
+#RUN curl -o /usr/bin/pandoc.gz \
+#         https://files.r-hub.io/pandoc/linux-64/pandoc.gz && \
+#    gzip -d /usr/bin/pandoc.gz && \
+#    curl -o /usr/bin/pandoc-citeproc.gz \
+#     https://files.r-hub.io/pandoc/linux-64/pandoc-citeproc.gz && \
+#    gzip -d /usr/bin/pandoc-citeproc.gz && \
+#    chmod +x /usr/bin/pandoc /usr/bin/pandoc-citeproc
```

Comment:
What pandoc version does arch come with? Maybe it is better to use these static ones?

Reply:
Arch packages tend to be updated pretty quickly so they are often at/close to the most recent stable versions, in this case, **pandoc 2.7.2** and **pandoc-citeproc 0.16.1.3**.

Reply:
Yeah, that's the thing, all other images have pandoc 1.x.x by default, and that's also what cran has. So maybe we should have 1.x.x here as well, and we can also have 2.x installed via sysreqs, if requested. This is to be worked out for the other images.

Reply:
Fair enough. I swapped out the pandoc versions. The command to installer the newer version of pandoc is still there in a comment just in case.